{"uid":"486fc79fd23d5a13","name":"User is able to login,this should fail","fullName":"tests.test_login.TestLogin#test_user_login","historyId":"88abe5aa9c6c37e78b845a216ec154ba","time":{"start":1684827368127,"stop":1684827402520,"duration":34393},"description":"User is able to login","descriptionHtml":"<p>User is able to login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:182:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:394:5\nelement.find/</<@chrome://remote/content/marionette/element.sys.mjs:134:16","statusTrace":"self = <tests.test_login.TestLogin object at 0xffff8e423c10>\n\n>   ???\n\n/Users/alya/Documents/selenium-ci-cd/tests/test_login.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/login_page.py:56: in login_existing_user\n    assert self.get_user_name() == \"Hi, \"+username\npages/nav_bar_page.py:25: in get_user_name\n    return self.get_text(self.LOGGED_IN_USER)\npages/base_page.py:33: in get_text\n    element: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"dbf71496-a870-4a4b-aac9-871078a6f3d8\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0xffff8e2d1ee0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:182:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:394:5\nE       element.find/</<@chrome://remote/content/marionette/element.sys.mjs:134:16\n\n/root/.local/share/virtualenvs/selenium-test-Ou9L0FGN/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":5,"retriesStatusChange":false,"beforeStages":[{"name":"setup","time":{"start":1684827363113,"stop":1684827368126,"duration":5013},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"description":"User is able to login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:182:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:394:5\nelement.find/</<@chrome://remote/content/marionette/element.sys.mjs:134:16","statusTrace":"self = <tests.test_login.TestLogin object at 0xffff8e423c10>\n\n>   ???\n\n/Users/alya/Documents/selenium-ci-cd/tests/test_login.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/login_page.py:56: in login_existing_user\n    assert self.get_user_name() == \"Hi, \"+username\npages/nav_bar_page.py:25: in get_user_name\n    return self.get_text(self.LOGGED_IN_USER)\npages/base_page.py:33: in get_text\n    element: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"dbf71496-a870-4a4b-aac9-871078a6f3d8\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0xffff8e2d1ee0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:182:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:394:5\nE       element.find/</<@chrome://remote/content/marionette/element.sys.mjs:134:16\n\n/root/.local/share/virtualenvs/selenium-test-Ou9L0FGN/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"User logs in","time":{"start":1684827368127,"stop":1684827402518,"duration":34391},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:182:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:394:5\nelement.find/</<@chrome://remote/content/marionette/element.sys.mjs:134:16\n\n","statusTrace":"  File \"/root/.local/share/virtualenvs/selenium-test-Ou9L0FGN/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/selenium-test/pages/login_page.py\", line 56, in login_existing_user\n    assert self.get_user_name() == \"Hi, \"+username\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/.local/share/virtualenvs/selenium-test-Ou9L0FGN/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/selenium-test/pages/nav_bar_page.py\", line 25, in get_user_name\n    return self.get_text(self.LOGGED_IN_USER)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/selenium-test/pages/base_page.py\", line 33, in get_text\n    element: WebElement = self.wait.until(\n                          ^^^^^^^^^^^^^^^^\n  File \"/root/.local/share/virtualenvs/selenium-test-Ou9L0FGN/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"User clicks login icon in nav bar","time":{"start":1684827368127,"stop":1684827368543,"duration":416},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Fill phone number","time":{"start":1684827368544,"stop":1684827369390,"duration":846},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"phone_number","value":"'01116888790'"}],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"User clicks start exploring","time":{"start":1684827369390,"stop":1684827369767,"duration":377},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"User fill password","time":{"start":1684827369767,"stop":1684827370610,"duration":843},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"password","value":"'test'"}],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"User clicks login button","time":{"start":1684827370610,"stop":1684827370948,"duration":338},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Return username when user logged in","time":{"start":1684827370948,"stop":1684827402516,"duration":31568},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:182:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:394:5\nelement.find/</<@chrome://remote/content/marionette/element.sys.mjs:134:16\n\n","statusTrace":"  File \"/root/.local/share/virtualenvs/selenium-test-Ou9L0FGN/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/selenium-test/pages/nav_bar_page.py\", line 25, in get_user_name\n    return self.get_text(self.LOGGED_IN_USER)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/selenium-test/pages/base_page.py\", line 33, in get_text\n    element: WebElement = self.wait.until(\n                          ^^^^^^^^^^^^^^^^\n  File \"/root/.local/share/virtualenvs/selenium-test-Ou9L0FGN/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[],"parameters":[{"name":"phone_number","value":"'01116888790'"},{"name":"password","value":"'test'"},{"name":"username","value":"'alyaothman'"}],"stepsCount":6,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[{"uid":"846f5d6d2f7e37fd","name":"stdout","source":"846f5d6d2f7e37fd.txt","type":"text/plain","size":162}],"parameters":[],"stepsCount":7,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"setup::0","time":{"start":1684827402551,"stop":1684827403362,"duration":811},"status":"passed","steps":[],"attachments":[{"uid":"85acd3fdc055a35f","name":"Screenshot","source":"85acd3fdc055a35f.png","type":"image/png","size":187735}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":false,"hasContent":true}],"labels":[{"name":"tag","value":"firefox"},{"name":"story","value":"Login functionality"},{"name":"tag","value":"@pytest.mark.usefixtures('setup')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"c84a2fb0e6ff"},{"name":"thread","value":"36-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"browser","value":"'firefox'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"633dbc1699862e61","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:182:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:394:5\nelement.find/</<@chrome://remote/content/marionette/element.sys.mjs:134:16","time":{"start":1684827325809,"stop":1684827362442,"duration":36633}},{"uid":"395bae0e985f412","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:182:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:394:5\nelement.find/</<@chrome://remote/content/marionette/element.sys.mjs:134:16","time":{"start":1684827279396,"stop":1684827316265,"duration":36869}},{"uid":"ccae26fd86970e58","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:182:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:394:5\nelement.find/</<@chrome://remote/content/marionette/element.sys.mjs:134:16","time":{"start":1684826898622,"stop":1684826935086,"duration":36464}},{"uid":"466c18449e572b61","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffff7d9ef9d0>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1684826614083,"stop":1684826614083,"duration":0}},{"uid":"7efcbb8eed3eb7c5","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffffab67a190>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1684826531273,"stop":1684826531273,"duration":0}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":4,"skipped":0,"passed":0,"unknown":0,"total":4},"items":[{"uid":"1f1b04088c420942","reportUrl":"../5/index.html#testresult/1f1b04088c420942","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:182:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:394:5\nelement.find/</<@chrome://remote/content/marionette/element.sys.mjs:134:16","time":{"start":1684826898622,"stop":1684826935086,"duration":36464}},{"uid":"a6e27f000ae1f5e1","reportUrl":"../4/index.html#testresult/a6e27f000ae1f5e1","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffff7d9ef9d0>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1684826614083,"stop":1684826614083,"duration":0}},{"uid":"db70a8e855bca114","reportUrl":"../3/index.html#testresult/db70a8e855bca114","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='selenium-hub', port=4444): Max retries exceeded with url: /session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffffab67a190>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1684826531273,"stop":1684826531273,"duration":0}}]},"tags":["firefox","@pytest.mark.usefixtures('setup')"]},"source":"486fc79fd23d5a13.json","parameterValues":["'firefox'"]}